server/test/unit/server/webservices/test_urls.py

```diff
@@ -46,16 +46,106 @@ def assert_url_match(expected_url, url_name, *args, **kwargs):
 
 
 class TestDjangoContentUrls(unittest.TestCase):
+    """
+    Test the matching of the content urls
+    """
+
+    def test_match_catalog_resource(self):
+        """
+        Test url matching for content_catalog_resource.
+        """
+        url = '/v2/content/catalog/mock-source/'
+        url_name = 'content_catalog_resource'
+        assert_url_match(url, url_name, source_id='mock-source')
+
+    def test_match_content_orphan_collection_view(self):
```

Comment:
I don't think you need '_view' in these. If anything it's a URL, but since the class says its a test for URLs I think naming the test the URLs name is best. In this case that would be 'content_orphan_collection'. Similar changes throughout the PR.